cmd/sks_nodepool.go

```diff
@@ -7,6 +7,7 @@ import (
 	"github.com/spf13/cobra"
 
 	egoscale "github.com/exoscale/egoscale/v2"
```

Comment:
perhaps you missed to remove this?
(if not, no worries, we don't have to migrate everything at once.)

Reply:
As the title of the PR specifies, I only migrated cluster creation but if preferred I can go further in this PR / following PRs :)